lib/npm-installation-manager.ts

```diff
@@ -137,13 +137,21 @@ export class NpmInstallationManager implements INpmInstallationManager {
 		return ((): boolean => {
 			let shasumProperty = "dist.shasum";
 			let cachedPackagePath = this.getCachedPackagePath(packageName, version);
-			let realShasum = this.$npm.view(`${packageName}@${version}`, shasumProperty).wait()[version][shasumProperty];
+			let packageInfo = this.$npm.view(`${packageName}@${version}`, shasumProperty).wait();
```

Comment:
in case the package is not in npm, this code will fail and we'll never get to the next if... or I'm totally missing something.

My test:
```
$ npm view app12354242
npm ERR! Windows_NT 6.3.9600
npm ERR! argv "c:\\Program Files\\nodejs\\node.exe" "c:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "view" "app12354242"
npm ERR! node v0.12.7
npm ERR! npm  v2.11.3
npm ERR! code E404

npm ERR! 404 Registry returned 404 for GET on https://registry.npmjs.org/app12354242
npm ERR! 404
npm ERR! 404 'app12354242' is not in the npm registry.
npm ERR! 404 You should bug the author to publish it (or use the name yourself!)
npm ERR! 404
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.

npm ERR! Please include the following file with any support request:
npm ERR!     d:\Work\nativescript-cli\npm-debug.log
```